testprogs/selpolltest.c

```diff
@@ -74,7 +74,7 @@ main(int argc, char **argv)
 	struct bpf_program fcode;
 	char ebuf[PCAP_ERRBUF_SIZE];
 	pcap_if_t *devlist;
-	int selectable_fd;
+	int selectable_fd = -1;
```

Comment:
This is presumably done to work around compilers that don't figure out that if neither `doselect` nor `dopoll` are set, `selectable_fd` isn't used.

It belongs in a separate change (which shouldn't require any review, so you might as well just commit it).

Reply:
fixed on main in fbb5e9ee